src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,618 @@
+                              return 1024;
+                          });
+                          mockStaticMockUpTest_MyClazz.when(() -> MockUpTest.MyClazz.staticMethod(anyInt())).thenAnswer(invocation -> {
+                              return 128;
+                          });
+                          assertEquals(1024, MockUpTest.MyClazz.staticMethod());
+                          assertEquals(128, MockUpTest.MyClazz.staticMethod(0));
+                      }
+                  }
+              
+                  public static class MyClazz {
+                      public static int staticMethod() {
+                          return 0;
+                      }
+              
+                      public static int staticMethod(int v) {
+                          return 1;
+                      }
+                  }
+              }
+              """));
+    }
+
+    @Test
+    void mockUpInstanceMethodTest() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.junit.Test;
+              import mockit.Mock;
+              import mockit.MockUp;
+              import static org.junit.Assert.assertEquals;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      new MockUp<MyClazz>() {
+                          @Mock
+                          public String getMsg() {
+                              return "mockMsg";
+                          }
+                          @Mock
+                          public String getMsg(String echo) {
+                              return "mockEchoMsg";
+                          }
+                      };
+                      assertEquals("mockMsg", new MyClazz().getMsg());
+                      assertEquals("mockEchoMsg", new MyClazz().getMsg("echo"));
+                  }
+              
+                  public static class MyClazz {
+                      public String getMsg() {
+                          return "msg";
+                      }
+              
+                      public String getMsg(String echo) {
+                          return echo;
+                      }
+                  }
+              }
+              """, """
+              import org.junit.Test;
+              import org.mockito.MockedConstruction;
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.AdditionalAnswers.delegatesTo;
+              import static org.mockito.ArgumentMatchers.*;
+              import static org.mockito.Mockito.*;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      MockUpTest.MyClazz mockObjMockUpTest_MyClazz = mock(MockUpTest.MyClazz.class, withSettings().defaultAnswer(CALLS_REAL_METHODS));
+                      doAnswer(invocation -> {
```

Comment:
If you have time, would be great to simply this to thenReturn

Reply:
Sure